src/lib/hooks/useUserTokenBalance.ts

```diff
@@ -0,0 +1,26 @@
+import { useQuery } from 'react-query';
+
+import { useZnsSdk } from './useZnsSdk';
+import { bigNumberToLocaleString } from '@zero-tech/zapp-utils/formatting/big-number';
+
+export const useUserTokenBalance = (account: string, paymentToken: string) => {
+	const sdk = useZnsSdk();
+
+	return useQuery(
+		['user-balance-token', account, paymentToken],
```

Comment:
`['user', 'balance', { account, paymentToken }]`

Reply:
updated in follow up pr